doc/go1.15.html

```diff
@@ -357,7 +357,10 @@ <h2 id="linker">Linker</h2>
   The linker now defaults to internal linking mode
   for <code>-buildmode=pie</code> on
   <code>linux/amd64</code> and <code>linux/arm64</code>, so these
-  configurations no longer require a C linker.
+  configurations no longer require a C linker. External linking
+  mode (which was the default in Go 1.14 for
+  <code>-buildmode=pie</code>) can still be requested with
+  <code>-ldflags=-linkmode=external</code> flag.
```

Comment:
In case anyone else is bothered by this: this seems to be a bug in Github as git doesn't show it.

```
$ git diff cf/cf-delegated-credentials cf/go-master --stat
 src/crypto/tls/common.go                     |  58 ---
 src/crypto/tls/conn.go                       |   4 -
 src/crypto/tls/delegated_credentials.go      | 530 ---------------------------
 src/crypto/tls/delegated_credentials_test.go | 446 ----------------------
 src/crypto/tls/generate_cert.go              |   6 -
 src/crypto/tls/handshake_client.go           |   9 +-
 src/crypto/tls/handshake_client_tls13.go     |  99 +----
 src/crypto/tls/handshake_messages.go         |  86 +----
 src/crypto/tls/handshake_messages_test.go    |   3 -
 src/crypto/tls/handshake_server.go           |  20 +-
 src/crypto/tls/handshake_server_tls13.go     |  89 +----
 src/crypto/tls/tls_test.go                   |   4 +-
 src/crypto/x509/x509.go                      |  15 -
```

Reply:
Might be due to rebases and merges — I think github mainly looks at the parent commit rather than recompute the diff.

Reply:
I"m closing this as well.